janusgraph-test/src/main/java/org/janusgraph/TestCategory.java

```diff
@@ -21,4 +21,5 @@
     public static final String UNORDERED_KEY_STORE_TESTS = "UNORDERED_KEY_STORE_TESTS";
     public static final String MEMORY_TESTS = "MEMORY_TESTS";
     public static final String PERFORMANCE_TESTS = "PERFORMANCE_TESTS";
+    public static final String CELL_TTL_TESTS = "CELL_TTL_TESTS";
```

Comment:
@farodin91 Hi, could you clarify some issues for me?

Why this category introduced? And why it is disabled by default?

I think `features.hasCellTTL()` more convenient when I run test from IDE, now IDE ignore this annotation and trying run these test on non supported backend

Reply:
I did this as a performance improvement. Before this change, maven would run start and stop phase for each test without doing anything.

I would like to move this behaviour into an extension for Junit5. To use feature flag from the backend directly to activate tests.

Reply:
Thanks

But `features.hasCellTTL()` options non conflict with TestCategory flag, this is second check for backend compatibility. May be restore part of code for simplify running test from IDE?

Reply:
I'm nearly finished the implementation to abort test based on backend features. https://github.com/farodin91/janusgraph/tree/feature-flag